src/tests/webkit/WebKit_Performance_001.test.js

```diff
@@ -1,35 +1,59 @@
-import { setWebKitUrl, calcAvgFPS, webKitBrowserActions } from '../../commonMethods/commonFunctions'
+import {
+  setWebKitUrl,
+  calcAvgFPS,
+  webKitBrowserActions,
```

Comment:
This function can be removed as it is not used

Reply:
right! thanks, will update.